docs/datasets/croatian.md

```diff
@@ -0,0 +1,454 @@
+# 🇭🇷 Croatian
+
+This is an overview of all the datasets used in the Croatian part of EuroEval. The
+datasets are grouped by their task - see the [task overview](/tasks) for more
+information about what these constitute.
+
+## Sentiment Classification
+
+### MMS-hr
+
+This dataset was published in [this paper](https://doi.org/10.48550/arXiv.2306.07902).
+The corpus consists of 79 manually selected datasets from over 350 datasets reported in the
+scientific literature based on strict quality criteria.
+
+The original dataset contains a single split with 77,594 Croatian samples. We use
+1,024 / 256 / 2,048 samples for our training, validation and test splits, respectively.
```

Comment:
Maybe we should mention that our sampling is stratified on the original datasets here.